### PR TITLE
php.ini has to be updated manually

### PR DESCRIPTION
**Pull request by pdelewski**
Installing extension via `pickle` requires updating `php.ini`

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

**Comment by chalin** [comment]
Preview: https://deploy-preview-2192--opentelemetry.netlify.app/docs/instrumentation/php/automatic/#setup

File: content/en/docs/instrumentation/php/automatic.md [hunk 1]
```diff
@@ -49,10 +49,17 @@ function ends it.
 
     * **pickle** can be used to install extensions that are available via http://pecl.php.net, however that's not the case
     for opentelemetry-php-instrumentation yet, so the only way for it is to install directly from source code.
-    Following command line shows how to do that using specific version of extension (1.0.0beta1 in this case).
+    Following command line shows how to do that using specific version of extension (1.0.0beta2 in this case).
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:52]:
```suggestion
    The following command line shows you how to do that using a specific version of the extension (1.0.0beta2 in this case):
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 2]
```diff
@@ -49,10 +49,17 @@ function ends it.
...
 
         ```console
-        $ php pickle.phar install --source https://github.com/open-telemetry/opentelemetry-php-instrumentation.git#1.0.0beta1
+        $ php pickle.phar install --source https://github.com/open-telemetry/opentelemetry-php-instrumentation.git#1.0.0beta2
+        ```
+
+        Add the extension to your php.ini
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:58]:
```suggestion
        Add the extension to your `php.ini` file:
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 3]
```diff
@@ -49,10 +49,17 @@ function ends it.
...
+
+        ```console
```
Comment (chalin) [inline content/en/docs/instrumentation/php/automatic.md:60]:
```suggestion
        ```ini
```

**Review by chalin — COMMENTED** [review]
See inline comments for suggested copyedits and a fixed code-block language. This is what the result looks like once included in the PR:

> image

Also: Prettier should be run over this file.

File: content/en/docs/instrumentation/php/automatic.md [hunk 4]
```diff
@@ -45,26 +45,39 @@ function ends it.
...
 
-        ```console
-        $ php pickle.phar install --source https://github.com/open-telemetry/opentelemetry-php-instrumentation.git#1.0.0beta1
-        ```
-   
-   *  **php-extension-installer**
-        ```console
-        $ install-php-extensions open-telemetry/opentelemetry-php-instrumentation@main
-        ```
+          ```console
```
Comment (pdelewski) [inline content/en/docs/instrumentation/php/automatic.md:60]: Seems that prettier messed up formatting

Comment (chalin) [reply content/en/docs/instrumentation/php/automatic.md:60]: Well yes and no. The formatting was a bit off / odd already.

Comment (chalin) [reply content/en/docs/instrumentation/php/automatic.md]: FYI, I committed a fix, which was just to unindent the two code blocks.

Comment (pdelewski) [reply content/en/docs/instrumentation/php/automatic.md]: thx.

**Review by chalin — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
Approving + merging as this is sourced from a maintainer!

**Pull request merged**